### PR TITLE
Add videos module 3, 4 and 5

### PR DESCRIPTION
**Pull request by flor14**

### CODE REVIEW

File: chapters/en/module3.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ When you are cloning a repository you are downloading a <code>.git</code> hidden
 </exercise>
 
 <exercise id='5' title="The staging area"  type='slides, video'>
-<slides source='module3/module3_03' shot='8' start='3:42' end='4:35'> </slides>
+<slides source='module3/module3_03' shot='8' start='14:35' end='19:59'> </slides>
```
Comment (joelostblom) [inline chapters/en/module3.md:137]: This video starts with you saying "Let's apply what we learned" from the following slide deck, maybe shift over 1-2 seconds?

This video also stops in the middle of a sentence

File: chapters/en/module3.md [hunk 2]
```diff
@@ -258,7 +258,7 @@ Have you include the README.md file in the staging area?
 </exercise>
 
 <exercise id='9' title='PUSHing changes from a remote repository'  type='slides, video'>
-<slides source='module3/module3_05' shot='8' start='3:42' end='4:35'> </slides>
+<slides source='module3/module3_05' shot='8' start='19:59' end='21:50'> </slides>
```
Comment (joelostblom) [inline chapters/en/module3.md:261]: This seems to not be the entire video

File: chapters/en/module3.md [hunk 3]
```diff
@@ -88,7 +88,7 @@ When you clone a Git repository from GitHub you get the hidden `.git` directory
 </exercise>
 
 <exercise id='3' title="Version control repositories"  type='slides, video'>
-<slides source='module3/module3_02' shot='8' start='3:42' end='4:35'> </slides>
+<slides source='module3/module3_02' shot='8' start='3:46' end='14:35'> </slides>
```
Comment (joelostblom) [inline chapters/en/module3.md:91]: This video stops before the "Lets apply what we learned" slide.

File: chapters/en/module3.md [hunk 4]
```diff
@@ -258,7 +258,7 @@ Have you include the README.md file in the staging area?
 </exercise>
```
Comment (joelostblom) [inline chapters/en/module3.md:258]: I think you missed one above `<exercise id='7' title='Committing changes to a local repository' type='slides, video'>` does not have the correct video added.

File: chapters/en/module3.md [hunk 5]
```diff
@@ -303,7 +303,7 @@ The HASH code it is automatically generated by Git each time you create a commit
...
 
 <exercise id='11' title='PULLing changes from a remote repository'  type='slides, video'>
-<slides source='module3/module3_06' shot='8' start='3:42' end='4:35'> </slides>
+<slides source='module3/module3_06' shot='8' start='21:51' end='27:10'> </slides>
```
Comment (joelostblom) [inline chapters/en/module3.md:306]: This starts in the middle of a slide.

**Review by joelostblom — CHANGES_REQUESTED** [review]
@flor14 I went through module 3 and there seems to be several video not starting and stopping at the correct time. I made comments for these but I didn't go through the other two modules yet, could you double check that all three modules' videos start and stop at the correct point in the netflix preview and I poing me so I can  review again?

**Comment by flor14** [comment]
Yes, I know. That is why I didn't assign you as reviewer. I will do this
tomorrow.

Thank you for your time and comments anyway 🙂

**Comment by joelostblom** [comment]
Whooops sorry about that, I didn't understand this was still a draft

**Comment by flor14** [comment]
@joelostblom here is my research about the copyright of the images (#5). There is only one image that could be problematic, but let me know your thoughts.

These are the images I have checked for copyright. Remember that all of them are included in the videos. 

* Allison Horst image (module5_05 slides)
She suggests donating to [indigenous woman who hikes](https://github.com/allisonhorst/stats-illustrations#is-the-artwork-useful-for-your-teaching-materials-i-would-be-so-grateful-if-you-would). I think that it could be a nice gesture to donate something. Also, she wants alt text in the slide, I added it.

* PHD comics (module3_01 slides)
This is the most complicated because we should send an email
image

* XKCD comics (module3_04 slides)
https://xkcd.com/1296/
This work is licensed under a [Creative Commons Attribution-NonCommercial 2.5 License](https://creativecommons.org/licenses/by-nc/2.5/).

* The Turing Way (module3_01 slides)
I cited the image as it is mentioned [here](https://zenodo.org/record/3332808#.Y0GvQ1LMJwo). It is a CC-BY license.

**Pull request merged**